include/rabit/serializable.h

```diff
@@ -9,7 +9,10 @@
 #include <vector>
 #include <string>
 #include "./internal/utils.h"
+
+#ifndef DMLC_IO_H_
 #include "../../dmlc-core/include/dmlc/io.h"
```

Comment:
This is a no go. If you do this you can just put the source of dmlc and rabit together in a single project. 
Correct way to do this would be to just include `dmlc/io.h` and to let cmake correctly determine include dirs.

Reply:
https://github.com/dmlc/rabit/pull/106/files